src/utils/Web3Converter.js

```diff
@@ -1,25 +1,49 @@
 import web3utils from 'web3/lib/utils/utils'
 import bs58 from 'bs58'
+import BigNumber from 'bignumber.js'
+
+const weiRate = 1000000000000000000
 
 class Web3Converter {
   /**
+   * @param n
+   * @param {Boolean} toWei
```

Comment:
don't specify types in your jsdoc
with flow you can do this right inside of your function signatures